src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Microsoft.DotNet.SourceBuild.SmokeTests.csproj

```diff
@@ -1,31 +1,93 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <!-- This project is built and run as part of the VMR and by sdk-diff-tests.yml and vmr-license-scan.yml. -->
+
   <PropertyGroup>
-    <TargetFramework>net9.0</TargetFramework>
-    <Nullable>enable</Nullable>
```

Comment:
Why remove this? We are still allowing nullable reference types: https://github.com/dotnet/installer/pull/19290/files#diff-96524322c45315bf2d420f760f663ae21977414d2835adddbe30b73aed834d37R25

Reply:
This got moved into the parent Directory.Build.props file so that it applies to all test projects by default.